### DISCUSSION TITLE
How to global search selected word with new dynamic search

### DISCUSSION THREAD

**Discussion by rcorre** [helix-editor/helix#11202] [Q&A] [answered]
As of https://github.com/helix-editor/helix/pull/9647, the `<space>/` global search is now interactive, which is awesome.
However, I frequently used to hit `<space>/<c-s>` to insert the word under the cursor before starting the search, as described here: https://github.com/helix-editor/helix/discussions/1933#discussioncomment-5381430.
The new picker opens immediately upon pressing `<space>/`, and hitting `<c-s>` just closes the picker. Is there a way to insert a word under the cursor, or pre-seed the search somehow?

**Comment by the-mikedavis** [comment, marked as answer]
You can copy the current selection (regex-escaped) to the search register `/` with `*` and then global search and press enter to search using the contents of the search register `/`. Or you can select the word under the cursor with `miw` and use `<C-r>.` in the picker which enters the contents of the current selections special register `.`.
